proto/messages.proto

```diff
@@ -50,9 +59,10 @@ message Response {
 
   oneof type {
     Error error = 2;
+    ChangesetError changeset = 3;
 
-    SessionResponse session = 3;
-    CreateSecretResponse create_secret = 4;
+    SessionResponse session = 4;
+    CreateSecretResponse create_secret = 5;
```

Comment:
Btw, in the future, we should avoid changing existing contracts. It is fine for now since we haven't shipped anything yet.

Reply:
yeah, since we're still developing with the feature flag, I'm okay with changing contracts